cmd/pkger/cmds/pack.go

```diff
@@ -123,7 +123,7 @@ func (e *packCmd) Flags() *flag.FlagSet {
 	return e.FlagSet
 }
 
-func Package(info here.Info, out string, decls parser.Decls) error {
+func Package(info here.Info, out string, subdir bool, decls parser.Decls) error {
```

Comment:
Changing this api would constitute a breaking change, so a different solution needs to be found.

Reply:
Fair enough. I'll remove that argument and check whether the directory matches info.Dir instead.